frontends/web/tests/setup.js

```diff
@@ -0,0 +1,5 @@
+import { createSerializer } from 'enzyme-to-json';
+ 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
```

Comment:
double check if this is necessary in addition to to snapshotSerializers in package.json
```
"snapshotSerializers": [
      "enzyme-to-json/serializer"
    ],
```